README.md

```diff
@@ -1 +1,72 @@
-# Awesome-Books
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books
+
+> week1 module 2 javascript project
+
+
+## Built With
+
+- HTML/JS 
+
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://becky449.github.io/Awesome-Books/)
+
```

Comment:
[OPTIONAL] This live demo link is broken. I recommend fixing this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.